modulefiles/wcoss2/wam-ipe.lua

```diff
@@ -35,20 +35,25 @@ load(pathJoin("pnetcdf", pnetcdf_ver))
 sp_ver=os.getenv("sp_ver") or "2.3.3"
 load(pathJoin("sp", sp_ver))
 
--- Look for libraries in "para"
-setenv("HPC_OPT", "/apps/ops/para/libs")
-prepend_path("MODULEPATH", "/apps/ops/para/libs/modulefiles/compiler/intel/19.1.3.304")
-prepend_path("MODULEPATH", "/apps/ops/para/libs/modulefiles/mpi/intel/19.1.3.304/cray-mpich/8.1.7")
+pio_ver=os.getenv("pio_ver") or "2.5.10"
+load(pathJoin("pio", pio_ver))
 
-esmf_ver=os.getenv("esmf_ver") or "8.4.0b08"
+esmf_ver=os.getenv("esmf_ver") or "8.4.1"
 load(pathJoin("esmf", esmf_ver))
 
 -- comio load occurs out of $HOMEwfs/modulefiles and is implicitly specified
 -- by build.ver as the version of COMIO that is checked out comes from build.ver.
-prepend_path("MODULEPATH", "../../../modulefiles")
-comio_ver=os.getenv("comio_ver") or "0.0.10"
+if os.getenv("HOMEwfs") then
+  prepend_path("MODULEPATH", pathJoin(os.getenv("HOMEwfs"), "modulefiles"))
+else
+  append_path("MODULEPATH", "/lfs/h1/swpc/wam/noscrub/adam.kubaryk/wam-ipe_workflow/modulefiles")
```

Comment:
I don't think NCO is going to want a personal account loaded into modules. You'd need to bump this into a group account, I believe.